pandas/io/sql.py

```diff
@@ -894,7 +902,12 @@ def insert(self, chunksize=None, method=None):
                 exec_insert(conn, keys, chunk_iter)
 
     def _query_iterator(
-        self, result, chunksize, columns, coerce_float=True, parse_dates=None
+        self,
+        result,
+        chunksize: Optional[str],
+        columns,
+        coerce_float: bool = True,
+        parse_dates=None,
```

Comment:
is this one correct? parse_dates

Reply:
I actually didn't touch this, but looking through these read_sql methods parse_dates defaults to None in most (I think all) of those

Reply:
maybe i pointed at the wrong one, this shouldn't be a bool type for parse_dates right? it takes a Optional[Dict[str, ....]]

Reply:
Ah yes it's one up from here.

Fixed (removed the annotation - sticking to doing plain bool/str in this PR)